src/test/java/tests/LoginTest.java

```diff
@@ -23,4 +25,21 @@ public void tc001_UserCanLogIntoRailwayWithValidUsernameAndPassword() {
         LogHelper.info("Verify that user can log into Railway with valid username and password");
         Assert.assertEquals(actual, "Welcome to Safe Railway", "Welcome message is not display like expected result");
     }
+
+    @Test(description = "User can't login with blank Username textbox")
+    public void tc02_UserCantLoginWithBlankUsernameTextbox() {
+        LogHelper.info("Click tab login");
+        loginPage.clickLoginTab();
+
+        LogHelper.info("Doesn't type any words into Username textbox but enter valid information into Password textbox" +
+                " and click Login button");
+        loginPage.login("", Constant.PASSWORD);
+
+        LogHelper.info("Get error message");
+        String actual = loginPage.getErrorMessage();
+
+        LogHelper.info("Verify that user can't login with blank Username textbox");
+        Assert.assertEquals(actual, errorMsg, "An error message is not display like expected result");
+
```

Comment:
redundant.

Reply:
@taile47711 you are missing this comment

Reply:
fixed